test/yaml-file.test.ts

```diff
@@ -6,14 +6,18 @@ import { YamlFile } from '../src/yaml-file';
 import { withTemporaryDirectory } from './testutil';
 
 describe('patch', () => {
-  function patchTest(patches: JsonPatch[], initialObj: any, assertObj: any) {
+  function patchTest(patches: JsonPatch[], initialObj: unknown, assertObj: unknown, updateObj?: unknown) {
```

Comment:
why unknown over any?

Reply:
Using `any` is much more dangerous than `unknown`. With `any` you can do anything with it, reverting to raw JavaScript behaviors (IOW no checks). With `unknown` it is saying that we don't want to disable checks, but we also aren't (yet) aware of the shape of this data.

If we do nothing with it directly but to pass it to other functions, then it makes little difference. But since code changes can later make a pass-through case into a use-it-now case one generally should avoid `any`.

Here's a more in-depth answer (not by me): https://dev.to/arikaturika/typescript-why-to-use-unknown-instead-of-any-41i8

Reply:
I should add that this change came from my configured [typescript-eslint](https://typescript-eslint.io/) static analysis rules. It's good practice, but I also generally avoid changing code unrelated to a PR in public PRs.

This is also what happened to the quotes in the markdown file. I have adjusted my local settings to be less aggressive when saving in non-local repos, sorry about that.